src/precompile.jl

```diff
@@ -9,10 +9,10 @@
     @precompile_all_calls begin
         # all calls in this block will be precompiled, regardless of whether
         # they belong to your package or not (on Julia 1.8 and higher)
-        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,DateTime,Bool,String,String,FixedDecimal{Int64,8}], ChunkedCSV.SkipContext())
-        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,DateTime,Bool,String,String,FixedDecimal{Int64,8}], ChunkedCSV.SkipContext(), _force=:parallel)
-        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,GuessDateTime,Bool,String,String,Int], ChunkedCSV.SkipContext())
-        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,GuessDateTime,Bool,String,String,Int], ChunkedCSV.SkipContext(), _force=:parallel)
+        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,DateTime,Bool,String,String,FixedDecimal{Int64,8}], ChunkedCSV.SkipContext(), buffersize=64)
+        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,DateTime,Bool,String,String,FixedDecimal{Int64,8}], ChunkedCSV.SkipContext(), _force=:parallel, nworkers=2, buffersize=64)
+        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,GuessDateTime,Bool,String,String,Int], ChunkedCSV.SkipContext(), buffersize=64)
+        ChunkedCSV.parse_file(IOBuffer(PRECOMPILE_DATA), [Int,Float64,Date,GuessDateTime,Bool,String,String,Int], ChunkedCSV.SkipContext(), _force=:parallel, nworkers=2, buffersize=64)
```

Comment:
why did this file change?

Reply:
Just to make sure that we precompile codepaths for parallel parsing by actually giving more than one chunk to work on

Reply:
make sense -- add a comment to the file saying that's what's happened? 
I think the different schema args are self-explanatory, but the `buffersize` less so